docker/start.sh

```diff
@@ -1,23 +1,10 @@
 #!/bin/bash
 
-DIR=$(pwd)
 COMMAND=${1}
 
 echo "K-Box start in progress..."
 
 case "${COMMAND}" in
```

Comment:
Probably at this point we can drop the switch statement on ${COMMAND}?

Reply:
probably yes, there is no reason to do have it